manifests/ChatQnA/chaqna-xeon-backend-server.yaml

```diff
@@ -36,11 +36,10 @@ apiVersion: v1
 metadata:
   name: chaqna-xeon-backend-server-svc
 spec:
-  type: NodePort
+  type: ClusterIP
   selector:
     app: chaqna-xeon-backend-server-deploy
   ports:
     - name: service
       port: 8888
       targetPort: 8888
-      nodePort: 30888
```

Comment:
UI will be using NodePort to access the backend-server.
Should update readme too.

Reply:
done